org.eclipse.xtext.ide/src/org/eclipse/xtext/ide/editor/folding/FoldingRangeKind.java

```diff
@@ -0,0 +1,18 @@
+/*******************************************************************************
+ * Copyright (c) 2021 TypeFox GmbH (http://www.typefox.io) and others.
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package org.eclipse.xtext.ide.editor.folding;
+
+/**
+ * @author Mark Sujew - Initial contribution and API
+ */
+public class FoldingRangeKind {
```

Comment:
please add  `@since 2.25` to new API classes

Reply:
Would it make sense to reuse the [org.eclipse.lsp4j.FoldingRangeKind](https://github.com/eclipse/lsp4j/blob/master/org.eclipse.lsp4j/src/main/java/org/eclipse/lsp4j/FoldingRangeKind.java) class instead of this class?

Reply:
@miklossy i assume the idea is to make the impl independent from LSP4J APIs. I like this approach a lot.